metrics/src/main/java/com/baeldung/metrics/aspectj/MetricsAspectJMain.java

```diff
@@ -0,0 +1,34 @@
+package com.baeldung.metrics.aspectj;
+
+import com.codahale.metrics.ConsoleReporter;
+import com.codahale.metrics.MetricRegistry;
+import com.codahale.metrics.SharedMetricRegistries;
+import java.io.PrintStream;
+import java.util.concurrent.TimeUnit;
+
+public class MetricsAspectJMain {
+    static final MetricRegistry registry = new MetricRegistry();
+
+    public static void main(String args[]) throws InterruptedException {
+        startReport();
+
+        ObjectRunner runner = new ObjectRunner();
+
+        for (int i = 0; i < 5; i++) {
+            runner.run();
+        }
+
+        Thread.sleep(3000L);
+    }
+
+    static void startReport() {
+        SharedMetricRegistries.add(ObjectRunner.REGISTRY_NAME, registry);
+
+        ConsoleReporter reporter = ConsoleReporter.forRegistry(registry)
+                .convertRatesTo(TimeUnit.SECONDS)
+                .convertDurationsTo(TimeUnit.MILLISECONDS)
+                .outputTo(new PrintStream(System.out))
+                .build();
+        reporter.start(3, TimeUnit.SECONDS);
```

Comment:
I'm not sure the _explanatory variable_ `reporter` adds much here... how about going from `.build()` to `.report()` fluently, rather than breaking the fluent chain with a temp variable that's only used straight afterwards.